### ISSUE TITLE
Can I use this lib in my Swift 4 project

### ISSUE THREAD

**Issue by rahul-inspired-iosdeveloper** [weilsonwonder/WWCalendarTimeSelector#80]
I want to use this lib in my swift 4 project. I have tried to install it thru pod installation but it showing too many errors, is it updated to swift 3 or 4 version.

**Comment by IOSDeveloperProgrammer** [comment]
@rahul-inspired-iosdeveloper  did you resolved this issue ? i was facing same issue and got many error at timeselector.swift file . please help me

**Comment by vineethmadathil** [comment]
Yes. You can. Follow this one https://github.com/weilsonwonder/WWCalendarTimeSelector/issues/73#issuecomment-398286492

**Comment by rahul-inspired-iosdeveloper** [comment]
> @rahul-inspired-iosdeveloper did you resolved this issue ? i was facing same issue and got many error at timeselector.swift file . please help me

Hi man 
sorry for late reply
I have fixed the issue by removing pod from project and added swift-4 files from GitHub.

**Comment by MithunMadhav1112** [comment]
@vineethmadathil I tried copy pasting that code as instructed. But still there are a lot of errors.
